bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java

```diff
@@ -241,7 +218,7 @@ public void start() {
             return;
         }
 
-        if (emailCapabilitiesProvider.canSendEmails()) {
+
```

Comment:
We need to have this check back now. Unless a phone can actually send out emails, we should not do anything!